vivo-project/VIVO-languages
# Removed dateTimeWithPrecision_fr_CA.ftl and added properties required to work with improved dateTimeWithPrecision.ftl

Fix to : https://jira.lyrasis.org/browse/VIVO-1877
Related PR : 
- https://github.com/vivo-project/Vitro/pull/203
- https://github.com/vivo-project/Vitro-languages/pull/39

###  What it does
Adds the properties required to make dateTimeWithPrecision.ftl core file universal (https://github.com/vivo-project/Vitro/pull/203). 
Also, removed unnecessary dateTimeWithPrecision_fr_CA.ftl file

### How to test it
Browse to Events > [some event] > Overview > date/time interval > add and check labels in the form.
Fix has been tested with fr_CA, en_CA, en_US, pt_BR, de_DE and es contexts.
